rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -1073,7 +1073,7 @@ namespace rsx
 					{
 						if (need_convert)
 						{
-							temp2.resize(out_pitch * std::max(convert_h, (u32)clip_h));
+							temp2.resize(out_pitch * (std::max(convert_h, (u32)clip_h) - 1) + (out_bpp * std::max(convert_w, (u32)clip_w)));
```

Comment:
What really happens in this case? I would have expected things to get clamped to out_pitch; is it not? Since the definition here is self-contradictory, the hardware would be wired to ignore one violation. So does it crop the image to the right or does it just write to the memory limit defined by the region?

Reply:
Afaik it it writes the data only to the memory limit defined by region. when the opposite happens it fills the rest of the dest memory with black.

Reply:
Does it crop the rows or overflow into the next one?

Reply:
crop.

Reply:
Btw, whis is this calculation even necessary? I mean the temp variables are system RAM so it shouldn't matter if we overallocate by a few bytes as long as we don't underallocate.

Reply:
right